### PR TITLE
Fix error output if no callback is provided

### PR DESCRIPTION
**Pull request by bambamboole**
This PR fixes the error output if no callback is provided.

Instead of nesting if conditions I went for a early return approach. I also provided a unit test for the cases.

If you want me to change anything, just drop a line :)

### CODE REVIEW

**Comment by maschmann** [comment]
Hi, thanks for contributing ;-) 
Normally I go for not having multiple returns within a method, but a single point of state return. There are some rare exceptions for this rule: e.g. if it has serious performance impacts due to heavy computational tasks. But if you have to consider such things, maybe the application design is flawed anyways :-D
In this particular case, it's OK to early return, though. Could you please fix the conflicts first?

**Comment by bambamboole** [comment]
Fixed the conflicts and removed the just merged not used use statement.

I do not want to sell you on early returns :) 
As you said, in this case, it really makes sense, because we just have different cases and having them out of mind via an early return reduces cognitive overhead.

**Comment by RoussKS** [comment]
Major fan of early returns here 😅😎

Doesn't matter, but helps for cleaner code imho

File: Tests/Asm/Ansible/Command/AnsiblePlaybookTest.php [hunk 1]
```diff
@@ -824,12 +818,6 @@ public function testExtraVars(): void
                 'input' => ['key1' => 'value1', 'key2' => 'value2'],
                 'expect' => '--extra-vars=key1=value1 key2=value2',
             ],
-            // Test valid JSON.
-            [
-                'input' => '{ "key1": "value1", "key2": "value2" }',
```
Comment (RoussKS) [inline Tests/Asm/Ansible/Command/AnsiblePlaybookTest.php:829]: @maschmann @bambamboole I think that the json or empty string vars tests have been removed?

**Pull request merged**